parsec/README.md

```diff
@@ -0,0 +1,101 @@
+# parsec
+
+A Flutter plugin for calculating math equations using C++ library.
+
+## Usage
+
+To use this plugin, add `parsec` as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
+
+### Example
+
+```dart
+import 'package:parsec/parsec.dart';
+
+void main() {
+    final Parsec _parsecPlugin = Parsec();
+    String equation = '5*5 + 5!';
+    dynamic _parsecResult;
+
+    try {
+        _parsecResult = await _parsecPlugin.eval(equation);
+    } catch (e) {
+        _parsecResult = e.toString();
+    }
+}
```

Comment:
Since this is just an example we should make it simple as possible

```suggestion
void main() {
    final Parsec parsec = Parsec();
    dynamic result;

    try {
        result = await _parsecPlugin.eval('5*5 + 5!');
    } catch (e) {
        result = e.toString();
    }
}
```